src/components/ZclCommandManager.vue

```diff
@@ -185,7 +187,13 @@ export default {
                   row.source == 'client') ||
                 (this.selectionServers.includes(this.selectedCluster.id) &&
                   row.source == 'server'))
-      && !this.selectionOut.includes(this.hashCommandIdClusterId(row.id, this.selectedCluster.id))
+      && !this.selectionOut.includes(this.hashCommandIdClusterId(row.id, this.selectedCluster.id)) ||
+        this.isCommandRequired(row) && (
+        (this.selectionClients.includes(this.selectedCluster.id) &&
+                  row.source == 'server') ||
+                (this.selectionServers.includes(this.selectedCluster.id) &&
+                  row.source == 'client'))
+      && !this.selectionIn.includes(this.hashCommandIdClusterId(row.id, this.selectedCluster.id))
```

Comment:
What is this above change doing?

Reply:
One of the bugs was that the command handling needs the warnings if you disable mandatory command. I don't know what the code does, but I clicked around there and it worked for me. So whatever the code does, is not doing too much damage. :)